ocamltest/ocaml_compilers.ml

```diff
@@ -40,30 +40,14 @@ class compiler
   method target = target
 
   method program_variable =
-<<<<<<< HEAD
-    Builtin_variables.program
-||||||| merged common ancestors
-    if Ocaml_backends.is_native host
-    then Builtin_variables.program2
-    else Builtin_variables.program
-=======
     if Ocaml_backends.is_native host && not Sys.win32
     then Builtin_variables.program2
     else Builtin_variables.program
->>>>>>> ocaml/5.1
```

Comment:
Why not resolve with `tip-jst`? It looks like we changed it to just `Builtin_variables.program` in https://github.com/ocaml-flambda/flambda-backend/pull/504.

Reply:
Ah yes, indeed.  Fixed